src/DatePicker/__tests__/DatePicker.test.tsx

```diff
@@ -20,11 +20,15 @@ describe('<DatePicker />', () => {
   describe('props', () => {
     it('passes props along to the StyledDatePicker', () => {
       const datePicker = shallow(
-        <DatePicker modifiers={modifiers} initialMonth={today} foo="bar" />
+        <DatePicker
+          modifiers={modifiers}
+          initialMonth={today}
+          className="lol"
```

Comment:
Replaced invalid props like `foo` with valid props (like `className`) throughout these specs now that TS is enforcing the prop types.